src/pages/sign-up/index.tsx

```diff
@@ -71,20 +70,26 @@ function SignUp(): JSX.Element {
     }
   };
 
-  const handleNextStep = (): void => {
-    setStep((prevStep) => prevStep + 1);
+  const handleSecondStep = (): void => {
+    setStep(STEPS.second);
+  };
+  const handleThirdStep = (): void => {
+    setStep(STEPS.third);
+  };
+  const handleFourthStep = (): void => {
+    setStep(STEPS.fourth);
   };
 
   return (
     <>
       <FlowHeader text={t('SignUp')} callback={handleBackStep} iconType="back" />
       <SignUpWrapper>
         <>
-          {step === 1 && <SignUpFirstForm onNext={handleNextStep} />}
+          {step === 1 && <SignUpFirstForm onNext={handleSecondStep} />}
           {step === 2 && (
-            <SignUpSecondForm role={(role as 'seeker') || 'caregiver'} onNext={handleNextStep} />
+            <SignUpSecondForm role={(role as 'seeker') || 'caregiver'} onNext={handleThirdStep} />
           )}
-          {step === 3 && <SignUpThirdForm onNext={handleNextStep} />}
+          {step === 3 && <SignUpThirdForm onNext={handleFourthStep} />}
           {step === 4 && <SignUpFourthForm onNext={handleSignUp} />}
```

Comment:
And now if we have STEPS constant lets use it and remove hardcoded values of step